lectures/ifp_egm.md

```diff
@@ -372,22 +373,21 @@ We use `static_argnums` to allow a recompile whenever `sizes` changes, since the
 K_egm_jax = jax.jit(K_egm, static_argnums=(3,))
 ```
 
-
 Next we define a successive approximator that repeatedly applies $K$.
 
 ```{code-cell} ipython3
 def successive_approx_jax(model,        
-            tol=1e-5,
-            max_iter=100_000,
-            verbose=True,
-            print_skip=25):
+                          tol=1e-5,
+                          max_iter=100_000,
+                          verbose=True,
+                          print_skip=25):
 
     # Unpack
-    constants, sizes, arrays = model
+    β, R, γ, s_size, y_size, s_grid, y_grid, P = model
     
-    β, R, γ = constants
-    s_size, y_size = sizes
-    s_grid, y_grid, P = arrays
+    constants = β, R, γ
+    sizes = s_size, y_size
+    arrays = s_grid, y_grid, P
```

Comment:
This is a small observation on the original code. Here we are packing `constants`, `sizes`, and `arrays` for the function `K_egm`, but later in `K_egm`, we unpack them again:

```
def K_egm(a_in, σ_in, constants, sizes, arrays):
    """
    The vectorized operator K using EGM.

    """
    
    # Unpack
    β, R, γ = constants
    s_size, y_size = sizes
    s_grid, y_grid, P = arrays
```

perhaps we can remove the packing and pass the `model` directly into  `K_egm` to remove this redundancy.

Reply:
Thanks @HumphreyYang . This is a good idea. I have thought and tried about this while coding but didn't find an elegant way to do then.